community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -120,6 +119,22 @@ public StateHandlingStatementOperations(
     }
 
     // <Cursors>
+
+    @Override
+    public Cursor<NodeItem> loadNodeById( KernelStatement statement, long nodeId ) throws EntityNotFoundException
```

Comment:
Actually they cannot be made protected since they are used at different level in the `Operations` stack.